tensorrt_llm/llmapi/tracer.py

```
@@ -1,9 +1,9 @@
 __all__ = ["enable_llm_tracer", "VizTracer", "log_sparse", "get_tracer"]
```

Ruff (0.15.0)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)